pycds/climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+            'display_name': 'Temperature Climatology (Min.)'
+        },
+        {
+            'name': 'Precip_Climatology',
+            'unit': precip_unit,
+            'standard_name': precip_standard_name,
+            'cell_method': 't: sum within months t: mean over years',
+            'description': 'Climatological mean of monthly total precipitation',
+            'display_name': 'Precipitation Climatology'
+        },
+    ]
+
+    for vs in variable_specs:
+        variable = session.query(Variable).filter(Variable.name == vs['name']).first()
+        if not variable:
+            vs.update(short_name='{0} {1}'.format(vs['standard_name'], vs['cell_method']),
+                      network_id=network.id)
+            session.add(Variable(**vs))
+    session.flush()
+
+
+# Names and widths of fields in flat source files. Copied from Faron's R code.
+# native_id is the native_id recorded in the database for the station
+# no history record is given but can use latest for the identified station
+field_names = ['native_id', 'assay_flag', 'station_name', 'elev', 'elevation_flag', 'long', 'lat',
+               '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', 'annual']
+field_widths = [8, 1, 12, 5, 1, 12, 12, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6]
+
+# Format string for module `struct`
+field_format = ' '.join(['{}s'.format(fw) for fw in field_widths])
+
+
+def load_pcic_climate_baseline_values(session, var_name, source):
+    """Load baseline values into the database.
+    Create the necessary variables and synthetic network if they do not already exist.
+
+    Args:
+        session (...): SQLAlchemy session for accessing the database
+
+        var_name (str): name of climate baseline variable for which the values are to be loaded
+
+        source (iterable): an interable that returns a sequence of fixed-width formatted ASCII lines
+            (strings) containing the data to be loaded; typically result of `file.readlines()`
+
+    Returns:
+        None
+    """
+
+    # Time (attribute) for each climate value should be the last hour of the last day of the month, year 2000.
+    baseline_year = 2000
+    def baseline_day(month):
+        """Return last day of month in baseline_year"""
+        return monthrange(baseline_year, month)[1]
+    baseline_hour = 23
+
+    def parse_line(line):
+        field_values = struct.unpack(field_format, bytes(line.rstrip('\n').encode('ascii')))
+        field_values = [fv.decode('ascii').rstrip('\0 ') for fv in field_values]  # struct.unpack creates null-terminated strings
+        return dict(zip(field_names, field_values))
+
+    create_pcic_climate_baseline_variables(session)
+    variable = session.query(Variable).filter(Variable.name == var_name).first()
+    if not variable:
+        raise InvalidVariableName("Climate variable named '{}' was not found in the database".format(var_name))
+
+    print('Loading...')
```

Comment:
I think that you'll probably want to use Python's logging module for anything that you have printed in here. It's a lot more flexible in that you can set various levels of output (critical, error, warning, info, debug) and can route different types of output to different destinations. I can point you to some of our other projects if you want guidance on how we have done this in the past.